sign_hab_imx8m.sh

```diff
@@ -28,18 +28,21 @@ if [ -z "$SRK_TABLE" ] || [ ! -f $SRK_TABLE ]; then
 	exit 1
 fi
 
+# retrieve the current script path in case it is executed from another location
+SCRIPT_PATH=$(dirname -- "${BASH_SOURCE[0]}")
```

Comment:
Can we simplify it to `dirname $0` ? I'd like it not to depend on bash (although I agree the script already depends on it.